### ISSUE TITLE
5.2.0-beta.2 AppImage does not start, chrome-sandbox seems to have wrong permissions

### ISSUE THREAD

**Issue by smrqdt** [meetfranz/franz#1477]
**Describe the bug**
Franz does not start when using the current AppImage.

Instead the following error message is shown:
`% ./franz-5.2.0-beta.2-x86_64.AppImage`
> [21423:0613/024732.992920:FATAL:setuid_sandbox_host.cc(157)] The SUID sandbox helper binary was found, but is not configured correctly. Rather than run without sandboxing I'm aborting now. You need to make sure that /tmp/.mount_franz-bGxm7V/chrome-sandbox is owned by root and has mode 4755.
> [1]    21423 trace trap (core dumped)  ./franz-5.2.0-beta.2-x86_64.AppImage

**To Reproduce**
1. Download 5.2.0-beta2 AppImage
2. `chmod +x franz-5.2.0-beta.2-x86_64.AppImage`
3. `./franz-5.2.0-beta.2-x86_64.AppImage`

**Expected behavior**
Franz should start.

**Desktop (please complete the following information):**
 - OS: Arch Linux, verified on Debian 9
 - Version 5.2.0 beta 2 (also on beta 1, last working AppImage is 5.1.0)

**Comment by Luxian** [comment]
Same issue in CentOS 7.

```
./franz-5.2.0-beta.2-x86_64.AppImage 
[8846:0614/084751.291462:FATAL:setuid_sandbox_host.cc(157)] The SUID sandbox helper binary was found, but is not configured correctly. Rather than run without sandboxing I'm aborting now. You need to make sure that /tmp/.mount_franz-QMQyFg/chrome-sandbox is owned by root and has mode 4755.
[1]    8846 trace trap (core dumped)  ./franz-5.2.0-beta.2-x86_64.AppImage
```

Running it with sudo makes it fail silently.

**Comment by smrqdt** [comment]
seems to be fixed in the final [v5.2.0 release](https://github.com/meetfranz/franz/releases/tag/v5.2.0) (at least for me)
